components/contribution-flow/StepDetails.js

```diff
@@ -43,7 +43,7 @@ const StepDetails = ({ onChange, data, collective, tier, showFeesOnTop, router,
   const { LoggedInUser } = useLoggedInUser();
 
   const minAmount = getTierMinAmount(tier, currency);
-  const hasQuantity = tier?.type === TierTypes.TICKET || tier?.type === TierTypes.PRODUCT;
+  const hasQuantity = (tier?.type === TierTypes.TICKET && !tier?.singleTicket) || tier?.type === TierTypes.PRODUCT;
```

Comment:
:ice_cream:  `tier` will exist if `tier.type` is set to TICKET

```suggestion
  const hasQuantity = (tier?.type === TierTypes.TICKET && !tier.singleTicket) || tier?.type === TierTypes.PRODUCT;
```